### PR TITLE
[Strings] Corrected typo

### PR DESCRIPTION
**Pull request by Timo4B**
moved the "," and deleted whitespace   #2902

### CODE REVIEW

File: concepts/strings/.meta/config.json [hunk 1]
```diff
@@ -1,5 +1,5 @@
 {
-  "blurb": "A  string (\"str\") is an immutable sequence of unicode code points. There is no separate \"character\" or \"rune\" type. Slicing copying, or concatenating will produce a new string.  Like any sequence, code points within a \"str\" can be referenced by 0-based index number and can be iterated over in a loop by using the \"for item in <str>\" construct.",
+  "blurb": "A  string (\"str\") is an immutable sequence of unicode code points. There is no separate \"character\" or \"rune\" type. Slicing, copying or concatenating will produce a new string. Like any sequence, code points within a \"str\" can be referenced by 0-based index number and can be iterated over in a loop by using the \"for item in <str>\" construct.",
```
Comment (BethanyG) [inline concepts/strings/.meta/config.json:2]:
```suggestion
  "blurb": "A  string (\"str\") is an immutable sequence of unicode code points. There is no separate \"char\" or \"rune\" type. Slicing, copying, or concatenating will produce a new string. Like any sequence, code points within a \"str\" can be referenced by 0-based index number and can be iterated over in a loop by using the \"for item in <str>\" construct.",
```

**Comment by Timo4B** [comment]
Sorry.

**Comment by Timo4B** [comment]
I'm new to github. I did not know it.

**Review by BethanyG — APPROVED** [review]
Thank you for this.  🌟

**Pull request merged**